discord/user/user.py

```diff
@@ -10,36 +10,21 @@
 
 class User(BaseUser):
 
-    __slots__ = ("_stored",)
+    _stored: bool = False
 
-    _stored: bool
-
-    def __init__(self, cache, payload: UserPayload):
-        super().__init__(cache, payload)
-        self._stored = False
-
-    @classmethod
-    def _from_user(cls, user: BU) -> BU:
-        self = super()._from_user(user)
-        self._stored = False
-        return self
-
-    def __repr__(self) -> str:
-        return f"{self.tag} ({self.id})"
+    @property
+    def mutual_guilds(self):
+        return
 
-    def __str__(self) -> str:
```

Comment:
Why removing this?

Reply:
I don't think that `User` needs a `__str__` magic method

Reply:
We use a `__str__` just to make sure we can always access the User tag easily

Reply:
But when a person prints a `User` object they probably expect the full representation not only the tag, if they want the full representation they will have to do `print(repr(user))` which I don't like, and they can access the tag using the `discriminator` attribute

Reply:
`BaseUser` has a `__str__` magic method which returns the username + discriminator

Reply:
Isn't the `tag` attribute `Username#discriminator`?